pyproject.toml

```diff
@@ -11,12 +11,14 @@ classifiers = [
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
     'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
     'Development Status :: 5 - Production/Stable',
     'Intended Audience :: Science/Research',
     'Topic :: Scientific/Engineering :: Astronomy',
 ]
 dependencies = [
   'asdf >= 2.12.1',
+  'asdf-standard @ git+https://github.com/braingram/asdf-standard.git@split_core'
```

Comment:
To be removed prior to merge and after https://github.com/asdf-format/asdf-standard/pull/422 is merged and released.